wt-status-serialize.c

```diff
@@ -255,7 +255,7 @@ static inline void wt_serialize_v1_changed(struct wt_status *s, int fd,
 	*p++ = '\0';
 	*p++ = '\n';
 
-	if (packet_write_gently(fd, begin, (p - begin)))
+	if (packet_write_gently(fd, begin, (p - begin), NULL))
```

Comment:
@jeffhostetler guaranteed the problem is here.

Reply:
yeah, you're probably segfaulting on that.